charts/sessionspaces/templates/clusterrolebinding.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "sessionspaces.fullname" . }}
+  namespace: kube-system
+subjects:
+- kind: ServiceAccount
+  name: {{ include "sessionspaces.serviceAccountName" . }}
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: {{ include "sessionspaces.fullname" . }}
```

This needs to match `metadata.name` in the associated `ClusterRole`
```suggestion
  name: {{ include "sessionspaces.serviceAccountName" . }}
```